instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -545,6 +547,8 @@ def __init__(
         http_capture_headers_server_request: list[str] | None = None,
         http_capture_headers_server_response: list[str] | None = None,
         http_capture_headers_sanitize_fields: list[str] | None = None,
+        exclude_receive_span: bool = False,
+        exclude_send_span: bool = False,
```

Comment:
Question: what do you think on adding something a list of possible values instead, e.g. `excluded_span: list[Literal["receive", "send"]] | None` ?

Reply:
For the sake of minimizing method arguments I guess, but really you'll still need to convert that into a boolean for each. Happy to make the change if you and @lzchen agree on it.

Reply:
I don't have a strong opinion on this.

Reply:
Alrighty, made the change